example/themes/business/src/components/BusinessController/index.tsx

```diff
@@ -97,6 +97,7 @@ export const BusinessControllerUI = (props: BusinessControllerParams) => {
             <Logo style={styles.logo}>
               <OIcon
                 url={optimizeImage(business?.logo, 'h_300,c_limit')}
+                src={business?.logo ? '' : theme?.images?.dummies?.businessLogo}
```

Comment:
improve this validation `src={!business?.logo && theme?.images?.dummies?.businessLogo}`